airflow/contrib/hooks/gcp_api_base_hook.py

```diff
@@ -57,7 +57,6 @@ class GoogleCloudBaseHook(BaseHook, LoggingMixin):
 
     JSON data provided in the UI: Specify 'Keyfile JSON'.
     """
-
```

Comment:
Unrelated white-space only change in a file that is otherwise untouched. Remove this change please.

Reply:
Sure. Automatic change by mistake.